privval/doc.go

```diff
@@ -6,16 +6,16 @@ FilePV
 
 FilePV is the simplest implementation and developer default. It uses one file for the private key and another to store state.
 
-SocketVal
+SignerValidatorEndpoint
```

Comment:
```suggestion
SignerValidatorServer
```

I haven't seen Endpoint suffix before.